Traceback (most recent call last):
  File "/home/yishido/self_supervied/test.py", line 214, in <module>
    main()
  File "/home/yishido/self_supervied/test.py", line 128, in main
    model = SimSiam(base_model).to(device)
  File "/home/yishido/self_supervied/test.py", line 62, in __init__
    self.encoder.fc,
  File "/home/yishido/.pyenv/versions/3.10.12/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1614, in __getattr__
    raise AttributeError("'{}' object has no attribute '{}'".format(
AttributeError: 'VisionTransformer' object has no attribute 'fc'